server/src/main/java/org/opensearch/wlm/QueryGroupLevelResourceUsageView.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.wlm;
```

Comment:
What does `wlm` stand for?

Reply:
I see from https://github.com/opensearch-project/OpenSearch/pull/14735/files#diff-f1059f230266d0d32d6e9ed96ba7be44008d74557715637b19926323199bd9f1 that it's `WorkloadManagement` but it looks like it's supposed to be in a plugin? Is this PR blocked on https://github.com/opensearch-project/OpenSearch/pull/14735 then?

Reply:
workload management

Reply:
>I see from [#14735 (files)](https://github.com/opensearch-project/OpenSearch/pull/14735/files#diff-f1059f230266d0d32d6e9ed96ba7be44008d74557715637b19926323199bd9f1) that it's WorkloadManagement but it looks like it's supposed to be in a plugin? Is this PR blocked on https://github.com/opensearch-project/OpenSearch/pull/14735 then?

No, this is not blocked on the Delete API PR, this code is supposed to sit in the core. 
wlm was introduced in core in [this PR](https://github.com/opensearch-project/OpenSearch/pull/14708/files#r1692213412)